securesystemslib/signer/_sigstore_signer.py

```diff
@@ -28,7 +28,10 @@
 class SigstoreKey(Key):
     """Sigstore verifier.
 
-    NOTE: unstable API - routines and metadata formats may change!
+    NOTE: The Sigstore key and signature metadata formats are not part of the
+    TUF specification and are not considered stable in securesystemslib. They
```

Comment:
maybe I shouldn't have specifically named TUF here but I guess the idea is clear: the formats are not "standardized" anywhere

Reply:
I'd include that Sigstore is uncertain until TAP-18 is accepted into the TUF spec. We don't have an in-toto spec for Sigstore keys specifically but I'm also of the opinion it doesn't entirely belong in the spec but rather in implementation docs. The spec / ITE only needs to identify the parameters that must be used to match the sigstore identity without worrying about other details.

There's generally more work we need to do unify the sslib specific documentation across in-toto and TUF, perhaps this is the home for it and both projects point here for what's supported once the signer interface is accepted.

Reply:
I don't think new key formats need to be added in the spec: even TAP-18 has no reason to be part of the spec as far as I'm concerned.  I am saying that adding support for new keys in a single implementation without clear community consensus is something we should be cautious about: I'm worried about both feature disparity and plain incompatibilities between different implementations of TUF (or intoto), e.g. python-tuf vs go-tuf.

I can try rephrasing this if it helps.

Reply:
I've removed the mention of TUF spec here